paper.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'scikit-COSMO: A Toolbox of Machine Learning Methods for Materials Science'
+tags:
+    - Python
+authors:
+    - name: Rose K. Cersonsky ^ [\url{mailto: rose.cersonsky@epfl.ch}]
+    orcid: 0000-0003-4515-3441
+    affiliation: 1
+    - name: Guillaume Fraux
+    orcid: 0000-0003-4824-6512
+    affiliation: 1
+    - name: Sergei Kliavinek
+    orcid: 0000-0001-8326-325X
+    affiliation: 1
+    - name: Alexander Goscinski
+    orcid: 0000-0001-8076-215X
+    affiliation: 1
+    - name: Benjamin A. Helfrecht
+    orcid: 0000-0002-2260-7183
+    affiliation: 1
+    - name: Michele Ceriotti ^ [\url{mailto: michele.ceriotti@epfl.ch}]
+    orcid: 0000-0003-2571-2832
+    affiliation: 1
+affiliations:
+    - name: Laboratory of Computational Science and Modeling(COSMO), École Polytechnique Fédérale de Lausanne(EPFL), Lausanne, Switzerland
+    index: 1
+date: 03 May 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`scikit-COSMO` (skcosmo) focuses on machine learning methods that are of particular use (but not limited
+                                                                                        to) the field of materials science and computational chemistry. Currently, many
+machine learning studies employ the foundational `scikit-learn` repository, a
```

Comment:
```suggestion
machine learning studies in computational chemistry employ the foundational `scikit-learn` repository, a
```

Reply:
More than just in computational chemistry. scikit-learn is used pretty universally

Reply:
Since the sentences around are specifically referring to this field, maybe something like:

> machine learning studies across fields of science employ the foundational `scikit-learn` repository, a

Reply:
The point I'm trying to make is the scikit-learn is intended for near-universal machine learning problems, and so it may not pay sufficient attention to some ML components as is necessitated within our field. Is that not coming across well?